tests/unit-tests/utils.spec.ts

```diff
@@ -2,82 +2,84 @@
 
 import { pick, mapValues, invert, pickBy, random } from 'src/common/utils';
 
-test('testing pick', () => {
-	const obj = { a: 1, b: 2, c: 3, d: 4 };
+describe('Testing all utility functions', () => {
```

Comment:
For the future again:
Surely you see that this change of adding descriptions is exactly the sort of completely unrelated change that should be done in another pull request.  These changes would even have almost no conflict with the other changes in this pull request, and if there are any minor conflicts they could easily be resolved.  This is what branches are for in git.

Reply:
Got it.